src/fable-fsharp/Replacements.fs

```diff
@@ -190,6 +193,8 @@ module Util =
                     |> makeCall com i.range (Fable.PrimitiveType (Fable.Number Int32))
                 Fable.Apply(op, [comp; makeConst 0], Fable.ApplyMeth, i.returnType, i.range)
                 |> Some
+            | Fable.Class None ->  // For enums only ("DeclaredType <enumName> Class null")
+                Fable.Apply(op, args, Fable.ApplyMeth, i.returnType, i.range) |> Some  
```

Comment:
Remove it and you will see in the tests :) 

Sorry. I made my comments in the issue https://github.com/fsprojects/Fable/issues/81 instead of here in the PR. Please review the issue.

But, in short, without this, `MyEnum.B >= a` will compile but `a >= MyEnum.B` won't.